src/db/shortcuts.ts

```diff
@@ -17,7 +17,7 @@ import type {
   Whereable,
   Table,
   Column,
-} from 'zapatos/schema';
+} from "zapatos/schema";
```

Comment:
what's driving the `'` to `"` change? it seems like in our codebase we prefer `'`?

Reply:
My vscode's "Format on save" setting. This is fixed now.